src/components/ChatLink/ChatLink.tsx

```diff
@@ -32,9 +35,7 @@ const ChatLink = () => {
   ];
 
   return (
-    <div>
-      {load ? <LoaderChatItem /> : <ChatItem messageItems={MOCK_MESSAGE} />}
-    </div>
+    <>{load ? <LoaderChatItem /> : <ChatItem messageItems={MOCK_MESSAGE} />}</>
```

Comment:
Unnecessary empty div. Instead of this better to use just `return load ? <LoaderChatItem/> : <ChatItem messageItems={MOCK_MESSAGE}/>`